src/C-interface/bml_allocate.c

```diff
@@ -12,7 +12,9 @@
 #include <stdlib.h>
 #include <string.h>
 
+#ifdef _OPENMP
 #include <omp.h>
+#endif
 
```

Comment:
Thanks!

It looks like as though this change broke the call at: https://github.com/lanl/bml/blob/master/src/C-interface/bml_allocate.c#L75. Could you `#ifdef` that call as well and set `nt = 1` in case the code is not built with OpenMP?

Thanks again for the PR!

Reply:
Oh, right. I didn't actually built it and I forgot to `grep` for function calls. It seems that was the only `omp_*` function call not guarded by an `#ifdef`. Fixed.